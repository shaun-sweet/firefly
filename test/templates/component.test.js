import Vue from 'vue'
import { shallow } from 'vue-test-utils'

import Component from '../../templates/component.vue'


describe('Component template', () => {
  it('should render the default component template', () => {
    const wrapper = shallow(Component)
    expect(wrapper.element).toMatchSnapshot();
  })
})
