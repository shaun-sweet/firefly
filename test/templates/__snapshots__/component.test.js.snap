// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component template should render the default component template 1`] = `<div />`;
